src/finufft.cpp

```diff
@@ -430,10 +428,10 @@ void deconvolveshuffle3d(int dir, FLT prefac, FLT *ker1, FLT *ker2, FLT *ker3, B
 
 // --------- batch helper functions for t1,2 exec: ---------------------------
 
-int spreadinterpSortedBatch(int batchSize, FINUFFT_PLAN p, CPX *cBatch)
+int spreadinterpSortedBatch(int batchSize, FINUFFT_PLAN p, CPX *fwBatch, CPX *cBatch)
```

Comment:
Passing fwBatch in explicitly, since it doesn't need to be part of the plan struct when DUCC is used.